packages/video_player/video_player/lib/video_player.dart

```diff
@@ -499,7 +499,12 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     } else if (position < const Duration()) {
       position = const Duration();
     }
-    await _videoPlayerPlatform.seekTo(_textureId, position);
+
+    try {
+      await _videoPlayerPlatform.seekTo(_textureId, position);
+    } catch (e) {
```

Comment:
Silently discarding all exceptions is not a fix, it's hiding the problem—and a variety of other possible problems as well, which is not okay.

`seekTo` isn't doing the same up-front `isInitialized` check that other methods are doing, which seems like the actual bug here. It looks like this flow throwing is a regression from https://github.com/flutter/plugins/pull/3727 (@cc @KyleFin)

Reply:
Thanks @stuartmorgan! Yes, this issue was mentioned in https://github.com/flutter/plugins/pull/3727#issuecomment-876879494. @jerryzhoujw let me know if there's anything I can do to help. Thanks!

Reply:
Instead of this change I believe you could just change line 397 to be
`if (value.position == value.duration && value.isInitialized) {`

Reply:
And unit tests could be similar to those added in https://github.com/flutter/plugins/pull/3727/files#diff-b67d4b0ee52dfea4c9c71082e12eca5459a174693c8d40577b0191d8ee498e34

Reply:
Thanks

Reply:
close this since https://github.com/flutter/plugins/pull/4300 much better.